api/valor_api/backend/core/llm_clients.py

```diff
@@ -11,6 +11,18 @@
 )
 from pydantic import BaseModel
 
+from valor_api.backend.core.llm_instructions_analysis import (
+    _generate_answer_relevance_verdicts_instruction,
```

Comment:
should these functions be private if they are imported here?

Reply:
What is the purpose of private functions? I've seen private functions as class functions, but I am defining these instructions outside of any class. 

I could make these functions class functions. DeepEval uses for example an AnswerRelevanceTemplate class and has the instructions as class functions with the @staticmethod decorator. I could just have an LLMInstructions class and make all of the instructions class functions of that class?